lib/matplotlib/collections.py

```diff
@@ -1171,29 +1171,52 @@ def __init__(self, segments,     # Can be None.
                  **kwargs
                  ):
         """
-        *segments*
-            a sequence of (*line0*, *line1*, *line2*), where::
+        Parameters
+        ----------
+        segments :
+            A sequence of (*line0*, *line1*, *line2*), where::
 
                 linen = (x0, y0), (x1, y1), ... (xm, ym)
 
             or the equivalent numpy array with two columns. Each line
             can be a different length.
 
-        *colors*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
+        colors : sequence, optional
+            A sequence of RGBA tuples (e.g., arbitrary color
             strings, etc, not allowed).
```

Comment:
Is this still true? I am wondering because the following
```python
import matplotlib.pyplot as plt
from matplotlib.collections import LineCollection
plt.ion()

coll = LineCollection([[[0, 0], [1, 1]], [[0, 1], [1, 0]]],
                      colors=["tab:gray", "tab:green"])

fig, ax = plt.subplots()
ax.add_collection(coll)

print("Click on the figure to change the colors")
plt.ginput(1)

coll.set_color(["tab:red", "tab:purple"])
```
seems to be fine, even though the colors are given as a list of arbitrary strings when instantiating the line collection.